_includes/sideblock.html

```diff
@@ -1,7 +1,13 @@
 <div class="sideblock-wrapper col-md-3 d-none d-lg-block">
   <div class="sideblock">
     {% if include.image %}
-    <img alt="" src="{{ include.image }}" style="max-width: 100%; margin-bottom: 10px">
+    {% if include.external_website %}
+    <a href="{{ include.external_website }}" target="_blank">
+    {% endif %}
+      <img alt="{{t.go_to_website}} {{ include.external_website }}" src="{{ include.image }}" style="max-width: 100%; margin-bottom: 10px">
```

Comment:
I'm not an expert in a11y but I think that an alt that says "go to website" without the actual URL is a bit counterintuitive... and then we are printing an empty variable. Probably it's safer to have a bigger if-then-else block, not really DRY but we know liquid is limited. What do you think?

Reply:
@libremente in case external_website is not defined yes, you're right! IMO it should be a mandatory field, anyway what about replacing {{ include.external_website }} with the platform's name? Result may be something like "Go to SPID website"

Reply:
Yes, probably a name is better in the alt.

Reply:
Fixed @libremente